CoreFoundation/Base.subproj/CFPlatform.c

```diff
@@ -628,6 +628,7 @@ static void *__CFTSDGetSpecific() {
 #if DEPLOYMENT_TARGET_MACOSX || DEPLOYMENT_TARGET_EMBEDDED || DEPLOYMENT_TARGET_EMBEDDED_MINI
     return pthread_getspecific(__CFTSDIndexKey);
 #elif DEPLOYMENT_TARGET_LINUX
+    __CFTSDInitialize();
```

Comment:
I think this actually belongs here:

https://github.com/apple/swift-corelibs-foundation/blob/ac7f8030ad381f0f3d97359f574a6c97a6cf66e1/CoreFoundation/Base.subproj/CFRuntime.c#L1080

where we can probably just remove the requirement for `DEPLOYMENT_RUNTIME_SWIFT` and make it `DEPLOYMENT_TARGET_LINUX`.